etna/transforms/missing_values/imputation.py

```diff
@@ -101,8 +145,9 @@ def __init__(
         self.default_value = default_value
         self.constant_value = constant_value
         self._strategy = ImputerMode(strategy)
-        self._fill_value: Optional[Dict[str, float]] = None
-        self._nan_timestamps: Optional[Dict[str, List[pd.Timestamp]]] = None
+        self._fit_segments: Optional[Sequence[str]] = None
+        self._nans_to_impute_mask: Optional[pd.DataFrame] = None
+        self._mean_imputer: SimpleImputerSubsegment = SimpleImputerSubsegment(strategy="mean", copy=False)
```

Comment:
Do we really need to save it even if we aren't using this strategy?

Reply:
No